### PR TITLE
Implement r2-4586: Add anniversary banner

### PR DESCRIPTION
**Pull request by mewtaylor**
This implements https://github.com/LLK/scratchr2/issues/4586. The only thing currently missing that needs to still be added is a link to the "Make a Card" activity (whenever you have a minute @seotts and @ntlrsk to add that let me know).

Currently looks like this in the following environments:
**12-column**
12-col

**8-column**
8-col

**6-column**
6-col

**4-column**
4-col

### CODE REVIEW

File: src/views/splash/birthday-banner/birthday-banner.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    <FlexRow className="banner-tenth-content">
+                        <div className="banner-tenth-content-text">
+                            <h2 className="h2 mod-white"><FormattedMessage id='splash.birthdayHeader' /></h2>
+                            <p className="p mod-white"><FormattedMessage id='splash.birthdayMessage' /></p>
+                        </div>
+                        <FlexRow className="banner-tenth-content-buttons">
+                            <a href="/studios/3959153/" className="button white mod-tenth-banner">
+                                <FormattedMessage id='splash.examples' />
+                            </a>
+                            <a href="#" className="button white mod-tenth-banner">
```
Comment (thisandagain) [inline src/views/splash/birthday-banner/birthday-banner.jsx]: I believe the activity is going to be a link to: `/projects/editor/?tip_bar=card` aka `/card`.

Comment (mewtaylor) [inline src/views/splash/birthday-banner/birthday-banner.jsx]: thanks :)

**Comment by mewtaylor** [comment]
On OSX 10.12, Chrome 57:
* Logged out, I see the banner
* `Make a Card` points to `/projects/editor/?tip_bar=card`
* `See example` points to `/studios/3959153/`
* Log in, banner still present
* Text still shows up in other language (though mostly as English for now)

**Comment by jwzimmer** [comment]
### Testing

* [x] iOS 10.3.1, Safari, iPad "LLK 1"
* (Downloaded Chrome and Firefox...)

**Pull request merged**